app/views/articles/_user_metadata.html.erb

```diff
@@ -1,122 +1,104 @@
-<style>
-  .user-metadata-details-inner a {
```

Comment:
what about this?

Reply:
What do you mean?

I basically got rid of profile colors for metadata - it was usually causing issues anyway (remember all the github PRs about text readability in different themes?) and instead I'll be introducing profile colors in other places (see that stripe at the top of the profile widget in sidebar?)

Reply:
oh cool! :D